### PR TITLE
Also lint .csv files for cds

### PR DESCRIPTION
**Pull request by mnkiefer**
- Support linting of `*.csv` files (for cds plugin) in VS Code
- Example:
Screenshot 2024-04-08 at 06 29 42

### CODE REVIEW

File: src/examples/index.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export async function loadExamples(): Promise<Record<string, Example>> {
       },
     ),
     ...Object.entries(
-      import.meta.glob("./**/*.{txt,js,cds}", {
+      import.meta.glob("./**/*.{txt,js,cds,csv}", {
```
Comment (ota-meshi) [inline src/examples/index.ts:41]: Do you also want to add csv files? It doesn't seem to be included in this PR.

Comment (mnkiefer) [reply src/examples/index.ts:41]: We do add _.csv_ files, but implicitly. ☺️ Our documentation rule examples (see the screenshot for the rule that lints csv) each *compute the link* to the playground, thereby providing the source files without us having to check them in here.

**Review by ota-meshi — APPROVED** [review]
